src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -62,14 +50,20 @@ public void storeMessage(Message message) {
       try {
         String idString = entity.getKey().getName();
         UUID id = UUID.fromString(idString);
+        UUID parent = UUID.fromString(idString);
```

Comment:
So for now, the parent of a comment is just itself? Or am I misunderstanding?

Reply:
My bad! It should get the parent property from the entity.